test/quantization/test_quant_api.py

```diff
@@ -90,6 +90,9 @@ def __init__(self):
         self.linear1 = torch.nn.Linear(64, 32).to(torch.float)
         self.linear2 = torch.nn.Linear(32, 64).to(torch.float)
 
+    def example_inputs(self):
+        return (torch.randn(1, 64),)
```

Comment:
nit: All the use cases unwrap the first index. Your call whether to pull this out of a singleton

Reply:
sg, I'll move the to(torch.float) here directly